hello-jam/TEAM

```diff
@@ -1,2 +1,2 @@
 massimo-zaniboni
-
+BaseMax
```

Comment:
since @massimo-zaniboni is the team captain, can I get a comment from them here okaying the PR?

I've been using the team captain as the person to check against to make sure the PR follows what the team wants to submit.

Reply:
yes, I confirm